py/frequenz/client/weather/_types.py

```diff
@@ -192,36 +192,51 @@ def to_ndarray_vlf(
             # get the location indexes of the proto for the filtered locations
             if locations:
                 for location in locations:
+                    found = False
                     for l_index, location_forecast in enumerate(
                         self._forecasts_pb.location_forecasts
                     ):
                         if location == Location.from_pb(location_forecast.location):
                             location_indexes.append(l_index)
+                            found = True
                             break
+                    if not found:
+                        # remember position of missing location
+                        location_indexes.append(-1)
```

Comment:
Because Python is so cool, it has `for/else` to do this:

```suggestion
                    for l_index, location_forecast in enumerate(
                        self._forecasts_pb.location_forecasts
                    ):
                        if location == Location.from_pb(location_forecast.location):
                            location_indexes.append(l_index)
                            break
                    else:
                        # remember position of missing location
                        location_indexes.append(-1)
```
(just wanted to mention, no action necessary)

Reply:
Wow, never heard of this.